app/bundles/CoreBundle/Views/Default/head.html.twig

```diff
@@ -2,7 +2,7 @@
     <meta charset="UTF-8" />
     <title>
         {% if headerTitle is defined and headerTitle is not empty %}
-            {{ headerTitle|replace({'<': ' <'})|striptags }} |
+            {{ headerTitle|replace({'<': ' <'})|striptags|raw }} |
```

Comment:
Can we use the `purify` filter instead of `raw`? It can be written by the users so this is open for XSS.

Reply:
https://github.com/mautic/mautic/pull/11974/commits/2a02f21431e2cb48a9b72dc834676f02301b7f2e